### DISCUSSION TITLE
Keep asking installing command line tool.

### DISCUSSION THREAD

**Discussion by lsl00** [PlayCover/PlayCover#576] [Q&A]
When I launch playcover,it always ask me to install xcode command line tool.However I have command line tools installed.
<img width="1012" alt="截屏2022-12-14 16 02 29" src="https://user-images.githubusercontent.com/39242557/207540247-513b8836-cc5a-4217-ad5d-cef9b16d96f6.png">
And when lanuch playcover from command line
<img width="873" alt="截屏2022-12-14 16 07 40" src="https://user-images.githubusercontent.com/39242557/207540490-687bf57f-a9f3-4588-8464-f518e46e01ef.png">

System version : macos ventura.
This problem occurs after updating system to ventura.
Does anyone konw how to fix this problem?

**Comment by amirsaam** [comment]
PlayCover requires Xcode Command Line Tools to be installed on macOS to work. You can see alternative ways to install it in this link: https://docs.playcover.io/getting_started/alt_xcode_cli_install
